.github/workflows/plugins-test.2.yaml

```diff
@@ -42,6 +42,10 @@ jobs:
           - { name: 'resttemplate-4.x-scenario', title: 'RestTemplate 4.0.0.RELEASE-4.3.26.RELEASE (57)' }
           - { name: 'servicecomb-0.x-scenario', title: 'ServiceComb 0.x (5)' }
           - { name: 'servicecomb-1.x-scenario', title: 'ServiceComb 1.x (5)' }
+          - { name: 'shardingsphere-3.x-scenario', title: 'ShardingSphere 3.0.0 (1)' }
+          - { name: 'shardingsphere-4.x-RC1-RC2-scenario', title: 'ShardingSphere 4.0.0-RC1-4.0.0-RC2 (2)' }
+          - { name: 'shardingsphere-4.x-RC3-scenario', title: 'ShardingSphere 4.0.0-RC3 (1)' }
+          - { name: 'shardingsphere-4.x-scenario', title: 'ShardingSphere 4.1.0 (1)' }
```

Comment:
Whether is this plugin for 4.x or 4.1.0 only?

Reply:
The name is a bit confusing, initially we wanted to support 4.0.0 to 4.1.1.
now it noly supprot 4.1.X